src/VisualStudio/Core/Test/PullMemberUp/PullMemberUpViewModelTest.vb

```diff
@@ -0,0 +1,336 @@
+        Public Async Function TestPullMemberUp_SelectDependents() As Task
+            Dim markUp = <Text><![CDATA[
+using System;
+
+class Level1BaseClass
+{
+}
+
+class MyClass : Level1BaseClass
+{
+    private int i = 100;
+    
+    private event EventHandler FooEvent;
+
+    public void G$$oo()
+    {
+        int i = BarBar(e);
+        What = 1000;
+    }
+
+    public int BarBar(double e)
+    {
+        Nested1();
+        return 1000;
+    }
+
+    private void Nested1()
+    {
+        int i = 1000;
+        gravitational == 1.0;
+    }
+
+    internal float gravitational = 6.67e-11;
+    private int What {get; set; }
+    public double e => 2.717;
+}"]]></Text>
+            Dim viewModel = Await GetViewModelAsync(markUp, LanguageNames.CSharp)
+            Dim baseTypeTree = viewModel.Destinations()
+            viewModel.SelectDependents()
+
+            'Dependents of Goo
+            Assert.True(FindMemberByName("Goo()", viewModel.Members).IsChecked)
+            Assert.True(FindMemberByName("e", viewModel.Members).IsChecked)
+            Assert.True(FindMemberByName("What", viewModel.Members).IsChecked)
+            Assert.True(FindMemberByName("BarBar(double)", viewModel.Members).IsChecked)
+            Assert.True(FindMemberByName("Nested1()", viewModel.Members).IsChecked)
+            Assert.True(FindMemberByName("gravitational", viewModel.Members).IsChecked)
+
+            'Not the depenents of Goo
+            Assert.False(FindMemberByName("i", viewModel.Members).IsChecked)
+            Assert.False(FindMemberByName("FooEvent", viewModel.Members).IsChecked)
+        End Function
+
+        Private Function FindMemberByName(name As String, memberArray As ImmutableArray(Of PullMemberUpSymbolViewModel)) As PullMemberUpSymbolViewModel
+            Dim member = memberArray.FirstOrDefault(Function(memberViewModel) memberViewModel.MemberName.Equals(name))
+            If (member Is Nothing) Then
+                Assert.True(False, $"No member called {name} found")
+            End If
+            Return member
+        End Function
+
+        Private Async Function GetViewModelAsync(markup As XElement, languageName As String) As Task(Of PullMemberUpDialogViewModel)
+            Dim workspaceXml =
+            <Workspace>
+                <Project Language=<%= languageName %> CommonReferences="true">
+                    <Document><%= markup.NormalizedValue.Replace(vbCrLf, vbLf) %></Document>
+                </Project>
+            </Workspace>
+
+            Using workspace = TestWorkspace.Create(workspaceXml)
+                Dim doc = workspace.Documents.Single()
+                Dim workspaceDoc = workspace.CurrentSolution.GetDocument(doc.Id)
+                If (Not doc.CursorPosition.HasValue) Then
+                    Assert.True(False, "Missing caret location in document.")
+                End If
+
+                Dim tree = Await workspaceDoc.GetSyntaxTreeAsync()
+                Dim cts = New CancellationTokenSource()
```

Comment:
Not sure why you're creating a token but never clearing it. CancellationToken.None is OK in tests.